generated/nidaqmx/system/device.py

```diff
@@ -1161,6 +1174,27 @@ def unreserve_network_device(self):
 
     # endregion
 
+    def get_ext_cal_last_date_and_time(device_name):
+        last_date_and_time = self._interpreter.get_ext_cal_last_date_and_time(device_name)
+
+        return datetime(
+            year = last_date_and_time[0],
```

Comment:
https://pep8.org/#other-recommendations

> Don’t use spaces around the = sign when used to indicate a keyword argument or a default parameter value.

I'm a bit surprised that ni-python-styleguide didn't warn about this, but maybe it doesn't enforce this rule.

Reply:
I think ni-python-styleguide skipped scanning files under `generated` folder as being configured in the `pyproject.toml`.

```
[tool.ni-python-styleguide]
extend_exclude = ".tox,docs,generated,src/codegen/metadata,src/codegen/templates,src/handwritten"
```

Reply:
Right, we don't run linters on generated code. I forgot that this file is generated

Also, we currently don't run linters on src/handwritten. We ought to fix that at some point, but not in this PR.